static/js/views/new-thing.js

```diff
@@ -67,71 +69,13 @@ class NewThing {
     const options = [];
     for (let capability of capabilities) {
       type = fluent.getMessageStrict(capability) || capability;
-      switch (capability) {
-        case 'OnOffSwitch':
-          cls = cls || 'on-off-switch';
-          break;
-        case 'MultiLevelSwitch':
-          cls = cls || 'multi-level-switch';
-          break;
-        case 'ColorControl':
-          cls = cls || 'color-control';
-          break;
-        case 'ColorSensor':
-          cls = cls || 'color-sensor';
-          break;
-        case 'EnergyMonitor':
-          cls = cls || 'energy-monitor';
-          break;
-        case 'BinarySensor':
-          cls = cls || 'binary-sensor';
-          break;
-        case 'MultiLevelSensor':
-          cls = cls || 'multi-level-sensor';
-          break;
-        case 'SmartPlug':
-          cls = cls || 'smart-plug';
-          break;
-        case 'Light':
-          cls = cls || 'light';
-          break;
-        case 'DoorSensor':
-          cls = cls || 'door-sensor';
-          break;
-        case 'MotionSensor':
-          cls = cls || 'motion-sensor';
-          break;
-        case 'LeakSensor':
-          cls = cls || 'leak-sensor';
-          break;
-        case 'PushButton':
-          cls = cls || 'push-button';
-          break;
-        case 'VideoCamera':
-          cls = cls || 'video-camera';
-          break;
-        case 'Camera':
-          cls = cls || 'camera';
-          break;
-        case 'TemperatureSensor':
-          cls = cls || 'temperature-sensor';
-          break;
-        case 'HumiditySensor':
-          cls = 'humidity-sensor';
-          break;
-        case 'Alarm':
-          cls = cls || 'alarm';
-          break;
-        case 'Thermostat':
-          cls = cls || 'thermostat';
-          break;
-        case 'Lock':
-          cls = cls || 'lock';
-          break;
-        case 'Custom':
-        default:
-          cls = cls || (capabilities.length > 1 ? '' : 'custom-thing');
-          break;
+
+      if (!cls) {
+        cls = getClassFromCapability(capability);
+
+        if (!cls && capabilities.length == 1) {
```

Comment:
This should be `<= 1` to match the previous logic.

Reply:
If assumed that if the length is `< 1`, the for loop will not be entered.
I omitted the condition because I considered it dead code.

Reply:
I assumed that if the length is < 1, the for loop will not be entered.
I omitted the condition because I considered it dead code.